src/applications/personalization/preferences/reducers/index.js

```diff
@@ -47,12 +47,19 @@ export default function preferences(state = initialState, action) {
 
       return {
         ...state,
-        dashboard: selectedBenefits,
+        dashboard: { ...selectedBenefits },
+        dashboardBackup: { ...selectedBenefits },
         userBenefitsLoadingStatus: LOADING_STATES.loaded,
       };
     }
     case SET_DASHBOARD_PREFERENCE: {
-      return _.set(`dashboard.${action.code}`, action.value, state);
+      const newState = { ...state };
+      if (action.value) {
+        newState.dashboard[action.code] = true;
+      } else {
```

Comment:
What's the reasoning for this change?

Reply:
Interesting... I saw `isEqualWith` was a function but didn't look at how it works...

Reply:
I mainly mention it to avoid potential merge conflicts, so nbd.